### PR TITLE
dxFieldChooser - SSR support, flex markup

### PR DESCRIPTION
**Pull request by AntonSermyazhko**

### CODE REVIEW

File: js/ui/pivot_grid/ui.pivot_grid.field_chooser.js [hunk 1]
```diff
@@ -334,21 +335,26 @@ var FieldChooser = BaseFieldChooser.inherit({
...
 
             that._renderArea($col1, "filter");
             that._renderArea($col1, "row");
             that._renderArea($col2, "column");
             that._renderArea($col2, "data");
         }
-        that.renderSortable();
 
-        that.updateDimensions();
+        if(hasWindow) {
```
Comment (nightskylark) [inline js/ui/pivot_grid/ui.pivot_grid.field_chooser.js]: Why don't move it to the `render` method?

File: js/ui/pivot_grid/ui.pivot_grid.field_chooser_base.js [hunk 2]
```diff
@@ -161,8 +162,12 @@ var FieldChooserBase = Widget.inherit(columnStateMixin).inherit(sortingMixin).in
 
     },
 
-    _renderContentImpl: function() {
-        this._headerFilterView.render(this.$element());
+    _initMarkup: function() {
+        this.callBase();
+
+        if(hasWindow) {
+            this._headerFilterView.render(this.$element());
```
Comment (nightskylark) [inline js/ui/pivot_grid/ui.pivot_grid.field_chooser_base.js]: Same here.

File: js/ui/pivot_grid/ui.pivot_grid.field_chooser.js (hunk 1 above)
Comment (AntonSermyazhko) [reply js/ui/pivot_grid/ui.pivot_grid.field_chooser.js]: Good idea! Will fix it..

File: js/ui/pivot_grid/ui.pivot_grid.field_chooser_base.js (hunk 2 above)
Comment (AntonSermyazhko) [reply js/ui/pivot_grid/ui.pivot_grid.field_chooser_base.js]: Ok

File: js/ui/pivot_grid/ui.pivot_grid.field_chooser.js [hunk 3]
```diff
@@ -347,14 +347,16 @@ var FieldChooser = BaseFieldChooser.inherit({
             that._renderArea($col2, "column");
             that._renderArea($col2, "data");
         }
+    },
 
-        if(hasWindow) {
-            that.renderSortable();
+    _render: function() {
+        var that = this;
```
Comment (nightskylark) [inline js/ui/pivot_grid/ui.pivot_grid.field_chooser.js]: The context caching is redundant here.

Comment (AntonSermyazhko) [reply js/ui/pivot_grid/ui.pivot_grid.field_chooser.js]: Ok, I've corrected it.

**Review by vconst — APPROVED** [review]

**Pull request merged**